package.json

```diff
@@ -37,6 +36,7 @@
     "chai-http": "^4.3.0",
```

Comment:
@urbanishimwe it's always a good practice to install the test-related dependencies as devDependencies so they are not piled up in production. We always strive for a lightweight prod environment.